src/rpc/rawtransaction.cpp

```diff
@@ -232,8 +241,9 @@ UniValue getrawtransaction(const UniValue &params, bool fHelp)
     }
 
     CTransactionRef tx;
+    int64_t txTime = GetTime(); // Will be overwritten by GetTransaction if we have a better value
```

Comment:
the above comment is not accurate, `txTime` is firstly set to 0 in `GetTransaction()` and then it will be set "fisrseen" time if the tx will be found in the mempool.

Reply:
fixed, thx!